recipes-tag/xpdan/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - pyyaml
     - scikit-beam
     - pyfai
+    - pyxdameraulevenshtein
```

Comment:
@CJ-Wright, why was this reverted? I think we decided to remove it in the earlier PRs (#729).

Reply:
Sorry I completely missed that

Reply:
No worries, we tracked this down and fixed via #737. The `xpd*` packages are compatible with the `*-2019-3.0` envs.

Reply:
Thank you so much!